kanopi/wp-vanilla-theme
# Define valid versions of node

## Description
> As a developer, I need to know which versions of node are compatible with the theme



## Steps to Validate
1. Install node 10.   Maybe nvm?
2. `npm install` should work
3. Install node 14
4. Delete `node_modules` folder
5. `npm install` should work
